[Vectorize] Add documentation for the Vectorize list-vectors operation (#24680)
cloudflare/cloudflare-docs: src/content/docs/vectorize/best-practices/list-vectors.mdx

---
title: List vectors
pcx_content_type: concept
sidebar:
  order: 5

---

The list-vectors operation allows you to enumerate all vector identifiers in a Vectorize index using paginated requests. This guide covers best practices for efficiently using this operation.

## When to use list-vectors

Use list-vectors for:

> - **Bulk operations**: When you need to process all vectors in an index
> - **Auditing**: To verify the contents of your index or generate reports
> - **Data migration**: When moving vectors between indexes or systems
> - **Cleanup operations**: To identify and remove outdated vectors

Reviewer: ```suggestion
- **Bulk operations**: To process all vectors in an index
- **Auditing**: To verify the contents of your index or generate reports
- **Data migration**: To move vectors between indexes or systems
- **Cleanup operations**: To identify and remove outdated vectors
```
